### PR TITLE
SPCTR-3542 - Updated the AI Assistant Sidebar UI.

### PR DESCRIPTION
**Pull request by bsf-zanev**
### Description

- Loaded the latest AI Assistant sidebar from the library.
- Integrated the filter to include the Spectra details in the sidebar.

### Types of changes

New feature (non-breaking change which adds functionality)

### How has this been tested?

Tested as shown in [this tasks](https://brainstormforce.atlassian.net/browse/SPCTR-3542) comments, and tested with integration of the filters in the Spectra branch.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-loader.php [hunk 1]
```diff
@@ -188,7 +188,25 @@ public function load_plugin() {
 				add_filter( 'ast_block_templates_disable', '__return_true' );
 			}
 
-			// Includes Zip AI library.
+			// Add the filter for the Zip AI Library and include it.
+			add_filter(
+				'zip_ai_collab_product_details',
+				function ( $product_details ) {
```
[the remark this answers is not in the document]
Comment (bsf-zanev) [reply classes/class-uagb-loader.php]: This has been done in [this commit](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3627/commits/da940fc72070b22d70e34787f7f83a11fec27794) - also the latest library updates were pulled in.

[the remark this answers is not in the document]
Comment (bsf-zanev) [reply classes/class-uagb-loader.php]: Updated the comment to match the function in [this commit](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3627/commits/39a3efb5288c3a4994d72c543d395c8082ce62cd).

File: classes/class-uagb-loader.php [hunk 2]
```diff
@@ -521,6 +523,27 @@ public function get_regenerate_assets_on_migration() {
 				\UAGB_Admin_Helper::update_admin_settings_option( '__uagb_asset_version', time() );
 			}
 		}
+
+		/**
+		 * Add the Zip AI Collab Prodcut Details.
```
Comment (adi3890) [inline classes/class-uagb-loader.php:528]:
```suggestion
		 * Add the Zip AI Collab Product Details.
```

Comment (bsf-zanev) [reply classes/class-uagb-loader.php]: Done in [this commit](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3627/commits/2f3e57eab1e20b43fd87733c2bae16b4ad4b1451) - thanks for spotting.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**